src/Views/HtmlViewBridge.js

```diff
@@ -773,7 +775,11 @@ HtmlViewBridge.prototype.raiseServerEvent = function (eventName) {
         argumentsArray[i] = arguments[i];
```

Comment:
yeah, they serve no purpose being passed to the client callback itself, I think you should remove them to stop people thinking it's really clever to be able to call them from their spaghetti mess.